### ISSUE TITLE
App purchased but still showing Trial Period message

### ISSUE THREAD

**Issue by caiopizzol** [ryouchinsa/Rectlabel-support#49]
I have purchased the application through App Store but It is not restoring my purchase neither allowing me to use it

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/33255434/44808923-d2d51900-aba3-11e8-9cb3-32b4e72566d3.png">

**Comment by ryouchinsa** [comment]
Thanks for writing the issue.
One of our users experienced the "damaged" message and could solve it. 

Could you follow these instructions at first?

1. Log out of the Mac App Store and iTunes.
2. Reboot your Mac.
3. Log back in to the Mac App Store.

If those instructions did not work, please follow these instructions.

1. Delete RectLabel via the Launchpad app
2. Log out of the Mac App Store and iTunes.
3. Reboot your Mac.
4. Log back in to the Mac App Store and install RectLabel.

Troubleshooting
https://rectlabel.com/help#troubleshooting

Basically this is the same problem as below.
https://github.com/ryouchinsa/Rectlabel-support/issues/48

**Comment by ryouchinsa** [comment]
Hi,

We are sorry that we could find a bug in our code.

- Fixed that for new users when you bought the Full App and restarted RectLabel, we were showing the "Buy Full App" dialog again.
We are recognizing whether the user is an existing user or a new user using the date when the user bought RectLabel.
When you launched RectLabel, the date was not taken from your app purchase receipt in the right timing.

Today we submitted the new update to Apple.
We will let you know when the new update version 2.07 is released.

We appreciate that you could report the problem.

**Comment by ryouchinsa** [comment]
The new update version 2.07 was released.
